charts/fluid/fluid/values.yaml

```diff
@@ -10,7 +10,9 @@ helmDriver: configmap
 
 crdUpgrade:
   enabled: true
-  image: fluidcloudnative/fluid-crd-upgrader:v1.0.0-985f833
+  # This sets the time-to-live (TTL) for crd-upgrade jobs. Default is 259200 seconds (3 days).
+  ttl: 259200 
```

Comment:
I suggest using `ttlSecondsAfterFinished` instead of `ttl`.

Reply:
ok.